tests/simulator/test_load_balancing.py

```
@@ -195,11 +178,25 @@ class LoadBalancingTest(SimulatorTest):
     def tearDown(self):
         ph.global_vars.sim = None
 
-    def run_sim(self, diags_dir, dic={}):
+    def run_sim(self, dic={}):
```

Ruff (0.15.4)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)